Improve detail links

georgf opened the pull request:
This addresses #350.

- Don’t add a “more details” link for SIMPLE_MEASURES_*, as the
probe-dictionary doesn’t have them.
- Directly link to the detailView for everything else (should be all
scalars & histograms).

At some point we’ll have to know the actual probe type (histogram,
scalar, …), but this seem to work for now.

georgf committed:
Remove only target="_blank" usage

This is the only place that is using target=“_blank”.
Erik caught that this is inconsistent and should probably just have
standard behaviour.

georgf committed:
Merge remote-tracking branch 'mozilla/gh-pages' into gh-pages

georgf committed:
Merge remote-tracking branch 'mozilla/gh-pages' into gh-pages

georgf committed:
Move mission control link further down

I assume that Mission Control is currently not as important as a link
target as HW report, Redash & Experiments Viewer.
Moving its link down accordingly.

georgf committed:
Merge remote-tracking branch 'mozilla/gh-pages' into gh-pages

georgf committed:
Add use counter dashboard

georgf committed:
Merge remote-tracking branch 'mozilla/gh-pages' into gh-pages

georgf committed:
Fix usecounters dashboard link

Publishing the usecounters dashboard from a separate dashboard broke
the “serve /index.html for /” behaviour.
Fixing the link here until i figure that out.

georgf committed:
Merge remote-tracking branch 'mozilla/gh-pages' into improve-detail-links

georgf committed:
Improve detail link to the probe-dictionary

- Don’t add a “more details” link for SIMPLE_MEASURES_*, as the
probe-dictionary doesn’t have them.
- Directly link to the detailView for everything else (should be all
scalars & histograms).

At some point we’ll have to know the actual probe type (histogram,
scalar, …), but this seem to work for now.

chutten commented on new-pipeline/src/dashboards.js:
```diff
@@ -1046,16 +1046,52 @@ function updateOSs() {
-                   "<i class=\"btn btn-outline-primary fa fa-info-circle\" " +
-                   "style='color: black' aria-hidden=\"true\">  " +
-                   "More Details </i></a>"
+  var html = descr;
```
Using `innerHTML`, you should wrap this in a tag. Or we should build the nodes using DOM APIs since metricURL contains user-supplied input, then we can avoid `innerHTML` entirely.

chutten reviewed (approved):
One smallish thing

georgf committed:
Use JQuery to create elements dynamically

Instead of using oversized strings to create elements, use JQuery
element creation.

georgf committed:
Also link for SIMPLE_MEASURES_*

I added the documented probes in simpleMeasurements too, so we can link
to them.

georgf commented:
@chutten Let's give this another round of review.
